checkov/cdk/checks/python/SecretManagerSecretEncrypted.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_149
+  name: Ensure that Secrets Manager secret is encrypted using KMS CMK
+  category: ENCRYPTION
+scope:
+  languages:
+    - python
+definition:
+  patterns:
+    or:
+      - pattern: aws_cdk.aws_secretsmanager.Secret(<ANY>)
+        conditions:
+          - not_pattern: aws_cdk.aws_secretsmanager.Secret(<ANY>, kms_key_id=$ARG , <ANY>)
+      - pattern: aws_cdk.aws_secretsmanager.Secret(<ANY>, kms_key_id=$ARG , <ANY>)
+        conditions:
+          - metavariable: $ARG
+            regex: (aws/)
```

Comment:
The $ARG is for the second pattern only? if yes, pls change it's name to ARG2 as it confusing..